build.gradle.kts

```diff
@@ -237,7 +238,7 @@ spotless {
     kotlin {
         target("**/*.kt")
         ktlint()
-        targetExclude("**/sentry-native/**")
+        targetExclude("**/sentry-native/**", "**/build/**")
```

Comment:
Had to add this exclusion otherwise spotless tries to format the generated code in the `build` folder. This was causing an issue since the generated accessors have an underscore (`_`) in the path name and this is was failing the check.